tests/manage/mcg/test_multi_region.py

```diff
@@ -62,6 +63,8 @@ def test_multiregion_bucket_creation(self, mcg_obj, multiregion_mirror_setup):
 
     @tier4
     @tier4a
+    @bugzilla('1827317')
+    @skipif_ocs_version("<4.5")
```

Comment:
Wouldn't be better to skip only 4.4? If I recall correctly then this worked in 4.3